operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -427,7 +427,7 @@ mod tests {
         let gdal_source = GdalSourceProcessor::<u8> {
             tiling_specification,
             meta_data: Box::new(create_ndvi_meta_data()),
-            phantom_data: Default::default(),
+            no_data_value: Some(0),
```

Comment:
you could also store the result of `create_ndvi_meta_data()` in a variable and get the no_data_value from the contained result_descriptor instead of repeating the concrete no_data_value in every test

Reply:
i did not change how the metadata stuff works.